Program.cs

```diff
@@ -10,7 +10,15 @@ internal class Program
     {
         static void Main(string[] args)
         {
-        Console.WriteLine("Ввведите трёхзначное число N:");
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Условия задачи:");
+            Console.WriteLine("");
+            Console.WriteLine("В трехзначном числе x зачеркнули последнюю цифру. " +
+                "Когда в оставшемся двузначном числе поменяли порядок цифр и приписали вычеркнутую цифру слева, получилось число n. " +
+                "По заданному числу n (ввод с клавиатуры и число единиц не равно нулю) найти число x.");
+            Console.WriteLine("");
+            Console.ResetColor();
+            Console.WriteLine("Ввведите трёхзначное число N:");
             int N= int.Parse(Console.ReadLine());
```

Comment:
В C# локальные переменные принято называть с маленькой буквы
```suggestion
            int n= int.Parse(Console.ReadLine());
```